app-android/src/androidTest/java/com/ustadmobile/port/android/view/VideoContentFragmentTest.kt

```diff
@@ -0,0 +1,120 @@
+package com.ustadmobile.port.android.view
+
+import android.Manifest
+import android.content.res.Configuration
+import androidx.core.os.bundleOf
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.rule.GrantPermissionRule
+import com.toughra.ustadmobile.R
+import com.ustadmobile.adbscreenrecorder.client.AdbScreenRecord
+import com.ustadmobile.core.view.UstadView.Companion.ARG_CONTAINER_UID
+import com.ustadmobile.core.view.UstadView.Companion.ARG_CONTENT_ENTRY_UID
+import com.ustadmobile.lib.db.entities.Container
+import com.ustadmobile.port.sharedse.contentformats.importContentEntryFromFile
+import com.ustadmobile.port.sharedse.util.UmFileUtilSe
+import com.ustadmobile.test.core.impl.CrudIdlingResource
+import com.ustadmobile.test.core.impl.DataBindingIdlingResource
+import com.ustadmobile.test.port.android.util.installNavController
+import com.ustadmobile.test.port.android.util.letOnFragment
+import com.ustadmobile.test.rules.ScenarioIdlingResourceRule
+import com.ustadmobile.test.rules.SystemImplTestNavHostRule
+import com.ustadmobile.test.rules.UmAppDatabaseAndroidClientRule
+import com.ustadmobile.test.rules.withScenarioIdlingResourceRule
+import kotlinx.coroutines.runBlocking
+import org.apache.commons.io.FileUtils.copyInputStreamToFile
+import org.hamcrest.core.AllOf.allOf
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.io.File
+import java.lang.Thread.sleep
+
+
+@AdbScreenRecord("Video Content Screen Test")
+@ExperimentalStdlibApi
+class VideoContentFragmentTest {
```

Comment:
This test needs to actually play the video and verify that the video plays (e.g. use onFragment to get access to the ExoPlayer instance and then get check that the status goes to playing.